test/tools/jsontests/Common.h

```diff
@@ -0,0 +1,55 @@
+#pragma once
+#include <test/tools/libtesteth/TestHelper.h>
+using namespace std;
+
+namespace dev
+{
+namespace test
+{
+enum class TestExecution
+{
+    REQUIRE_OPTION_ALL,
+    REQUIRE_OPTION_ALL_NOT_REFILLABLE,
+    NOT_REFILLABLE,
+    STANDARD
+};
+
+
+template <class T>
+class StateTestFixtureBase
+{
+public:
+    StateTestFixtureBase(TestExecution const _execFlag)
+    {
+        T suite;
+        if ((_execFlag == TestExecution::REQUIRE_OPTION_ALL_NOT_REFILLABLE ||
+                _execFlag == TestExecution::NOT_REFILLABLE) &&
+            (Options::get().fillchain || Options::get().filltests))
+            BOOST_FAIL("Tests are sealed and not refillable!");
+
+        string casename = boost::unit_test::framework::current_test_case().p_name;
+        boost::filesystem::path suiteFillerPath = suite.getFullPathFiller(casename).parent_path();
+        static vector<string> const timeConsumingTestSuites{
```

Comment:
Still it would be easier to maintain, if these constants are somewhere at the top of the file